sass/_themes.scss

```diff
@@ -45,19 +46,19 @@ $COLOR_GREEN_1: #4bb4b3;
  *****************************
 \*****************************/
 
-$color-black: #0e0e0e;
```

Comment:
@maryiafrantsava-deriv could you revert this back to lower case, these colors are not in capital letters in this section

Reply:
@akmal-binary do you mean all these colors?
<img width="320" alt="Screenshot 2021-09-15 at 11 38 36" src="https://user-images.githubusercontent.com/88426229/133400603-24acec84-35c7-49a7-a6dc-2cdd42f8d853.png">

Reply:
Fixed